ride_share.rb

```diff
@@ -0,0 +1,103 @@
+# Step 3: Make the data structure!
+
+# Setup the entire data structure:
+# based off of the notes you have above, create the
+# and manually write in data presented in rides.csv
+# You should be copying and pasting the literal data
+# into this data structure, such as "DR0004"
+# and "3rd Feb 2016" and "RD0022"
+
+# the dates needed a bit of tweeking to be apropriately ordinalized.
+
+# based on layering (nesting) method 3:
+all_rides = {
+  :DR0001 => [
```

Comment:
Don't forget, when you use symbols as a hash key, you can write lines like this as `DR0001: [`